src/test/mir-opt/enum_opt.trunc.EnumSizeOpt.64bit.diff

```diff
@@ -0,0 +1,42 @@
+- // MIR for `trunc` before EnumSizeOpt
++ // MIR for `trunc` after EnumSizeOpt
+  
+  fn trunc() -> () {
+      let mut _0: ();                      // return place in scope 0 at $DIR/enum_opt.rs:62:16: 62:16
+      let mut _1: Truncatable;             // in scope 0 at $DIR/enum_opt.rs:63:7: 63:12
+      let mut _2: Truncatable;             // in scope 0 at $DIR/enum_opt.rs:64:7: 64:32
+      let mut _3: [u8; 1024];              // in scope 0 at $DIR/enum_opt.rs:64:22: 64:31
+      let mut _4: Truncatable;             // in scope 0 at $DIR/enum_opt.rs:65:7: 65:32
+      let mut _5: [u8; 4096];              // in scope 0 at $DIR/enum_opt.rs:65:22: 65:31
+      scope 1 {
+          debug a => _1;                   // in scope 1 at $DIR/enum_opt.rs:63:7: 63:12
+      }
+  
+      bb0: {
+          StorageLive(_1);                 // scope 0 at $DIR/enum_opt.rs:63:7: 63:12
+          Deinit(_1);                      // scope 0 at $DIR/enum_opt.rs:63:15: 63:29
+          discriminant(_1) = 0;            // scope 0 at $DIR/enum_opt.rs:63:15: 63:29
+          StorageLive(_2);                 // scope 1 at $DIR/enum_opt.rs:64:7: 64:32
+          StorageLive(_3);                 // scope 1 at $DIR/enum_opt.rs:64:22: 64:31
+          _3 = [const 0_u8; 1024];         // scope 1 at $DIR/enum_opt.rs:64:22: 64:31
+          Deinit(_2);                      // scope 1 at $DIR/enum_opt.rs:64:7: 64:32
+          ((_2 as B).0: [u8; 1024]) = move _3; // scope 1 at $DIR/enum_opt.rs:64:7: 64:32
+          discriminant(_2) = 1;            // scope 1 at $DIR/enum_opt.rs:64:7: 64:32
+          StorageDead(_3);                 // scope 1 at $DIR/enum_opt.rs:64:31: 64:32
+          _1 = move _2;                    // scope 1 at $DIR/enum_opt.rs:64:3: 64:32
+          StorageDead(_2);                 // scope 1 at $DIR/enum_opt.rs:64:31: 64:32
+          StorageLive(_4);                 // scope 1 at $DIR/enum_opt.rs:65:7: 65:32
+          StorageLive(_5);                 // scope 1 at $DIR/enum_opt.rs:65:22: 65:31
+          _5 = [const 0_u8; 4096];         // scope 1 at $DIR/enum_opt.rs:65:22: 65:31
+          Deinit(_4);                      // scope 1 at $DIR/enum_opt.rs:65:7: 65:32
+          ((_4 as C).0: [u8; 4096]) = move _5; // scope 1 at $DIR/enum_opt.rs:65:7: 65:32
+          discriminant(_4) = 2;            // scope 1 at $DIR/enum_opt.rs:65:7: 65:32
+          StorageDead(_5);                 // scope 1 at $DIR/enum_opt.rs:65:31: 65:32
+          _1 = move _4;                    // scope 1 at $DIR/enum_opt.rs:65:3: 65:32
+          StorageDead(_4);                 // scope 1 at $DIR/enum_opt.rs:65:31: 65:32
+          _0 = const ();                   // scope 0 at $DIR/enum_opt.rs:62:16: 66:2
+          StorageDead(_1);                 // scope 0 at $DIR/enum_opt.rs:66:1: 66:2
+          return;                          // scope 0 at $DIR/enum_opt.rs:66:2: 66:2
+      }
+  }
```

Comment:
There's no actual difference here... does the pass not do anything?

Reply:
This specific diff was intended to show that it does not modify if there are overflows.
I left a comment above the enum that was intended to be not transformed, is there another place I should put it?